test_files/values.alp

```diff
@@ -15,6 +18,20 @@ let test_atom = :nope
 let test_bool = true
 let test_chars = c"x"
 let test_tuple = (100, "100")
+let test_unit = ()
+
+let wrapped_int = AnInt 41
```

Comment:
@lepoetemaudit: I tried writing
````
````
but then I got `{error,non_literal_value, ...`.  Perhaps we should allow arbitrary names in here, and resolve the references in alpaca_ast_gen. What do you think?